src/procstat.c

```diff
@@ -1058,7 +1099,7 @@ void fuse_setattr(fuse_req_t req, fuse_ino_t ino, struct stat *attr, int to_set,
 	}
 
 	/* only support control files  */
-	if (!item_type_control(item)) {
+	if (!item_type_control(item) && !fuse_inode_to_file(ino)->writer) {
```

Comment:
why do you need to check for existence of writer here?

Reply:
in order to prevent set attribute on non writable files